aait/backend/group-2B/Application/DTOs/PostDtos/Validators/UpdatePostDtoValidator.cs

```diff
@@ -0,0 +1,49 @@
+using FluentValidation;
+using SocialSync.Application.Contracts.Persistence;
+using SocialSync.Application.DTOs.PostDtos;
+
+namespace SocialSync.Application.DTOs.PostDtos.Validators;
+public class UpdatePostDtoValidator : AbstractValidator<UpdatePostDto>
+{
+    private IUserRepository _userRepository;
```

Comment:
Here, it would be better if the _userRepository and _postRepository marked as readonly to prevent unintentional updates and to clarify the intention.

Reply:
Fixed.